Bio/motifs/__init__.py

```diff
@@ -24,16 +24,18 @@ def create(instances, alphabet=None):
 
 
 def parse(handle, format):
-    """Parse an output file from a motif finding program.
+    """
+    Parse an output file from a motif finding program.
```

Comment:
Please leave the triple-quote and the first line of the docstring on the same line:

```python
def parse(handle, format):
    """Parse an output file from a motif finding program.
```